Makefile

```diff
@@ -46,9 +46,11 @@ run: setup-cfg-dir
 	docker run -it --rm \
 		-v $(PWD):/shared \
 		-v /dev/iotlab:/dev/iotlab \
+		-v /dev/ttyCN:/dev/ttyCN \
```

Comment:
@aabadie  mentioned it, but what happens when the host doesn't have /dev/ttyCN ?

Reply:
What happens is that the Docker daemon creates a folder `/dev/ttyCN` and mounts it !
Of course this is not clean. What about moving the symbolic link `ttyCN` to the `/dev/iotlab`folder? 
I can give it a shot if you agree...

Reply:
Yeah I think the best would be to put ttyCN in /dev/iotlab. 
in udev SYMLINK+="ttyCN iotlab/ttyCN" would create both symbolic links, I think

Reply:
I would prefer adding some Makefile logic that sets a variable with mapping if requested by the user:
```
make USE_CN=1 integration-test
```

In the makefile, test that USE_CN is equal to 1, default value is 0:
```mk
USE_CN ?=0

DOCKER_CN_MAPPING=
ifeq (1,$(USE_CN))
  DOCKER_CN_MAPPING = -v /dev/ttyCN:/dev/ttyCN
endif
```

And then in the integration-test target:
```mk
docker run -it --rm \
 		-v $(PWD):/shared \
 		-v /dev/iotlab:/dev/iotlab \
		$(DOCKER_CN_MAPPING) \
```

Not tested, but you have the idea.

Reply:
@aabadie I see the idea, the logic is actually in the variable `CONTROL_NODE_TYPE`. I am pushing a fix.